scipy/fftpack/tests/test_real_transforms.py

```diff
@@ -3,9 +3,11 @@
 from os.path import join, dirname
 
 import numpy as np
-from numpy.testing import assert_array_almost_equal, assert_equal
+from numpy.testing import (assert_array_almost_equal, assert_equal,
+                           assert_raises)
```

Comment:
This is just being fixed in gh-7777, so better ``from pytest import raises as assert_raises``